### PR TITLE
Restyle transactionActions and unify icons with other icons

### PR DESCRIPTION
**Pull request by ninabreznik**
First and second task from https://github.com/ethereum/browser-solidity/issues/824
screen shot 2018-02-11 at 12 34 30 am

### CODE REVIEW

**Comment by ninabreznik** [comment]
@yann300 This one is also ready to merge...

**Comment by yann300** [comment]
the `0 pending transaction` label is fixed?

**Comment by ninabreznik** [comment]
@yann300 Fun fact. I was now checking this pull request for like an hour, re-adding but by bit to see where the pending transactions text disappeared and couldn't find it. And then I checked the live version on remix.ethereum.org and saw that it also doesn't appear there. So it looks like the problem is not in this PR, but we lost the label before somewhere...

**Comment by yann300** [comment]
oh ok right :) thanks for checking that

File: src/app/tabs/styles/run-tab-styles.js [hunk 1]
```diff
@@ -170,13 +175,21 @@ var css = csjs`
...
   .transactionActions {
-    float: right;
+    display: flex;
+    width: 70px;
+    justify-content: space-between;
+    border: 1px solid ${styles.rightPanel.runTab.additionalText_Color};
+    padding: 5px;
+    border-radius: 3px;
+}
+
```
Comment (yann300) [inline src/app/tabs/styles/run-tab-styles.js]: ... standard checking should not pass here....

Comment (yann300) [reply src/app/tabs/styles/run-tab-styles.js]: ah it is because this is inside the string

Comment (ninabreznik) [reply src/app/tabs/styles/run-tab-styles.js]: @yann300 All good?

**Pull request merged**